dolphinscheduler-ui/src/common/types.ts

```diff
@@ -19,6 +19,7 @@ import { Component } from 'vue'
 export type ITaskState =
   | 'SUBMITTED_SUCCESS'
   | 'RUNNING_EXECUTION'
+  | 'STOP'
```

Comment:
hi @imizao  It's best to sort by ID. Can you put STOP after PAUSE? Of course it won't have any impact if you don't change it.